lms/djangoapps/instructor/services.py

```diff
@@ -63,7 +63,13 @@ def delete_student_attempt(self, student_identifier, course_id, content_id):
 
         if student:
             try:
-                enrollment.reset_student_attempts(course_id, student, module_state_key, delete_module=True)
+                enrollment.reset_student_attempts(
+                    course_id,
+                    student,
+                    module_state_key,
+                    delete_module=True,
+                    requesting_user=requesting_user,
```

Comment:
I think this was added as the second to last param, and not the last param.  Assuming this is a bug, can you determine why there wasn't a test failure?